BoxBrowseSDK/BoxBrowseSDK/BOXItemsViewController.h

```diff
@@ -143,4 +143,13 @@
  */
 - (BOOL)itemsViewControllerShouldShowCloseButton:(BOXItemsViewController *)itemsViewController;
 
+/**
+ *  A custom title for the Close button used to dismiss the view controller
+ *  By default, the title is "Close".
+ *
+ *  @param itemsViewController The instance of BOXItemsViewController calling this method.
+ *  @return The string to be displayed for the button.
+ */
+- (NSString *)itemsViewControllerCloseButtonTitle:(BOXItemsViewController *)itemsViewController;
```

Comment:
All of the methods in the protocol start with itemsViewController - for the sake of consistency / less autocompletion pollution it might be best to keep it this way?